### ISSUE TITLE
Default mail templates getting stuffed with <br> in place of new lines

### ISSUE THREAD

**Issue by tegola** [laravel/framework#26844]
- Laravel Version: 5.7.17
- PHP Version: 7.1.19

### Description:

I'm having a strange issue where a standard Laravel notification mail is being stuffed with extraneous line breaks. It looks like all the line breaks from the html files are converted to `<br>`s.

For example, the header component template:
```html
<tr>
    <td class="header">
        <a href="{{ $url }}">
            {{ $slot }}
        </a>
    </td>
</tr>
```

after compiling, becomes:

```html
<tr>
<br><td class="header" style="...">
<br><a href="http://localhost:8888" style="..."><br>
            Laravel<br></a><br>
</td>
<br>
</tr>
```

(I have removed the inlined css for readability)

The end result is this:

![example](https://user-images.githubusercontent.com/1868590/49966076-5a5e5200-ff1f-11e8-8a24-e1e6eef96a10.png)

At this point I don't know if it's a bug. I'm using Laravel 5.7 but it's not a clean install. There is little or no customization and I have always followed the upgrade guides and commits from version to version.

Any idea?

**Comment by driesvints** [comment]
Can you first please try one of the following support channels? If you can actually identify this as a bug, feel free to report back.

- [Laracasts Forums](https://laracasts.com/discuss)
- [Laravel.io Forums](https://laravel.io/forum)
- [Discord](https://discordapp.com/invite/KxwQuKb)
- [Larachat](https://larachat.co)
- [IRC](https://webchat.freenode.net/?nick=laravelnewbie&channels=%23laravel&prompt=1)
- [StackOverflow](https://stackoverflow.com)

**Comment by tegola** [comment]
I already tried Discord and StackOverflow without success.

**Comment by driesvints** [comment]
@tegola can you post the link to your stackoverflow post?

**Comment by tegola** [comment]
Sure:
https://stackoverflow.com/questions/53752385/laravel-notification-template-includes-extraneous-line-breaks-br

**Comment by driesvints** [comment]
It's not a day old. Give it a bit of time and try one of the other channels as well. The forums are usually a good place to get help.

**Comment by tegola** [comment]
Hello, it's been almost two months and this issue is still unresolved. I couldn't find a solution and the stack overflow post is still there unanswered.

Is there a chance of somebody having a look at it?

**Comment by driesvints** [comment]
@tegola I'm afraid that I won't be able to help you with this. I sincerely doubt that this is a bug with the framework since this is the first time I've ever seen this problem. It's best that you continue to try on a support channel.

**Comment by tegola** [comment]
@driesvints I'll try, thanks.

I think moving everything to a fresh project would be faster, even if it's bothersome and not bug-free.

**Comment by dfoxx** [comment]
@tegola Had a similar issue, turned out I was using `Blade::setEchoFormat('nl2br(e(%s, true))');` in the `AppServiceProvider` to pretty print text with line breaks: https://stackoverflow.com/a/56527018/1543490

**Comment by tegola** [comment]
@dfoxx Sorry for the late reply. `Blade::setEchoFormat()` was the first thing I removed when I was having this issue, and it didn't help. Also moving to a new version of Laravel didn't help, so I "fixed" it by making my own templates instead of using mailables.

I have now moved everything on the latest Laravel (6.4.x) and I can't replicate the bug, everything's working normally.

Thanks for you help anyway!
